packages/lit-dev-content/site/tutorials/content/tooltip/07.md

```diff
@@ -0,0 +1,58 @@
+You've polished the tooltip's behavior with animations. Now it's time to ensure
+great performance. Since a tooltip displays only when the user interacts with its
+target, it doesn't need to be created until then. If a page uses a lot of
+tooltips, which is not uncommon, adding this optimization can speed up initial
+page rendering by a meaningful amount.
+
+To do this, you'll create a function users will call with a tooltip target
+element and a callback that will render tooltip content on demand when
+its needed. Add a `static` method to the `SimpleTooltip` class called
```

Comment:
New paragraph before `Add` since it's directly actionable it could use its own paragraph start.

Reply:
Done